vms/platformvm/validator_set_property_test.go

```diff
@@ -373,9 +373,11 @@ func addPrimaryValidatorWithoutBLSKey(vm *VM, data *validatorInputData) (*state.
 }
 
 func internalAddValidator(vm *VM, signedTx *txs.Tx) (*state.Staker, error) {
+	vm.ctx.Lock.Unlock()
 	if err := vm.Network.IssueTx(context.Background(), signedTx); err != nil {
 		return nil, fmt.Errorf("could not add tx to mempool: %w", err)
```

Comment:
Are we okay returning when the mutex is unlocked on error here?

Reply:
No we need to release the lock on error, fixed.